app/controllers/posts_controller.rb

```diff
@@ -725,9 +724,13 @@ def markdown(post)
   # We can't break the API for making posts. The new, queue supporting API
   # doesn't return the post as the root JSON object, but as a nested object.
   # If a param is present it uses that result structure.
-  def backwards_compatible_json(json_obj, success)
+  def backwards_compatible_json(json_obj)
```

Comment:
Removed the 2nd parameter `success` since we can extract the information from `json_obj`.